### ISSUE TITLE
Copy the testing and linting pipelines/GH actions

### ISSUE THREAD

**Issue by jeremyharisch** [kyma-project/lifecycle-manager#1307]
**Acceptance Criterias**
- Copy and adept E2E tests from Kyma-CLI regarding Module Scaffolding
- Copy and adapt pipelines from Kyma CLI (linting, unit-tests trigger, etc.)
    - Exclude Release pipelines, will be covered in follow-up issue
- Make sure Pipelines are set as required

**Comment by janmedrek** [comment]
Blocked by: https://github.com/kyma-project/modulectl/issues/9

**Comment by jeremyharisch** [comment]
Already covered in the initial implementation
